build.sbt

```diff
@@ -92,7 +92,7 @@ lazy val V = new {
   val vaultDriver = "6.2.0"
   val micrometer = "1.11.11"
 
-  val nimbusJwt = "10.0.0"
+  val nimbusJwt = "9.37.3"
```

Comment:
it would be nice to know what is wrong with the 10.0.0 version
it might be the update of the BouncyCastle library